index.html

```diff
@@ -1369,27 +1368,27 @@ <h3>Hashing Algorithms</h3>
       <p class="advisement">
         When in doubt in selection of a hashing algorithm, consult the
         <a href="https://csrc.nist.gov/projects/hash-functions">
-          NIST documentation</a> related to hash function selection,
+          NIST documentation</a> related to hash function selection.
+        For new implementations, consider
         <a href="https://csrc.nist.gov/publications/detail/fips/202/final">
-          SHA-3 as described in FIPS 202</a>
-        should be strongly considered for new implementations
+          SHA-3 as described in FIPS 202</a>.
       </p>
     </section>
 
     <section class="informative">
       <h3>Randomness</h3>
-      When making an implemention carefully consider how you are sourcing
+      When making an implemention, carefully consider how you are sourcing
       random numbers. Consult
       <a href="https://datatracker.ietf.org/doc/html/RFC4086">RFC 4086:
         Randomness Requirements for Security</a>
-      when selecting an approach to get random bits, and pay careful attention
-      to the platform and any underlying hardware that may be in use as multiple
+      when selecting an approach to get random bits. Pay careful attention
+      to the platform and any underlying hardware that may be in use. Multiple
       attacks have been performed in the wild due to improper selection of
       random values in key material and other aspects of cryptography.
     </section>
 
     <section class="informative">
-      <h3>Zero Knowledge Proofs</h3>
+      <h3>Zero-Knowledge Proofs</h3>
```

Comment:
this is not a comment on @clehner 's excellent editorial work.

Why does the DID Implementation Guide talk about zero-knowledge proofs at all? Perhaps to the extent that an issuer may wish to use a DID Document to publish the verification key for a zero-knowledge proof–capable VC, but beyond that, the information in this section seems to relate little to DIDs

Reply:
DID documents are to contain such crypto material as may be necessary to communicate with the entity identified by that DID. DID method specifiers and implementers may use ZKPs for these purposes, which possibility I think makes this section just as relevant as any of the others here.

Reply:
Really, my concern with the section is not that it talks about ZKPs and how they may be used with DIDs, my concern is that it doesn't.

The guidance here is to 1) use proper curves for ZKPs, 2) not use ZKP proof formats tied to specific ledger technology, and 3) not store schemas on ledgers.
What does this guidance have to do with DIDs?

Reply:
@brentzundel -- This should perhaps become a new issue of its own?